lambda-notes/src/components/ListView.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import Note from './Note';
+import './App.css';
+import './ListView.css';
+
+const ListView = (props) => {
+    return (
+        <div className='list-container'>
+            <div className='list-header'>
+                <h4>Your Notes:</h4>
+            </div>
+            <div className='note-list'>
+                {props.notes.map(note => <Note key={note.id} note={note} />)}
```

Comment:
Check the name of this prop, looks like you called it something else in App.js :)